### PR TITLE
Port the last test cases into the /test-cases

### PR DESCRIPTION
**Pull request by fruch**
@aleksbykov @yarongilor @amoskong @ShlomiBalalis @fgelcer 
Please take a look at test-case you were involved in, to review we didn't dropped anything important.

I've only run `test-cases/manager/manager-regression-multiDC-set-distro.yaml` and it passed provision, and manager installation (failed on non related issue #919)  
while doing so fixes some issues with the new config using multi-region setups

@roydahan, I think we should sit down 1:1 on this like we did last time.

### CODE REVIEW

File: sdcm/sct_config.py [hunk 1]
```diff
@@ -731,6 +737,13 @@ class SCTConfiguration(dict):
                 be provided by the test suite infrastructure.
                 multiple commands can passed as a list"""),
 
+        dict(name="stress_cmd_sst3_verify_delete", env="SCT_STRESS_CMD_SST3_VERIFY_DELETE", type=str_or_list,
```
Comment (roydahan) [inline sdcm/sct_config.py]: @fruch this "param" and all others that looks like it above, shouldn't be in our SCT config.
(Or at least I don't think they should).
They are specific to tests (and also were changed lately).

File: test-cases/performance/perf-regression-latency-1TB.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+user_prefix: 'perf-regression-latency'
+failure_post_behavior: destroy
+space_node_threshold: 644245094
+
+round_robin: 'true'
+append_scylla_args: '--blocked-reactor-notify-ms 4'
+
+send_email: true
+email_recipients: ['qa@scylladb.com', 'dev-int@scylladb.com']
```
Comment (roydahan) [inline test-cases/performance/perf-regression-latency-1TB.yaml]: I think we should change all places with dev-int to rnd-int

File: test-cases/sct-cases/dns-cluster-5min.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# TODO: need to qualify
```
Comment (roydahan) [inline test-cases/sct-cases/dns-cluster-5min.yaml]: With regards to our conversation about jobs, this one also need a job.

File: test-cases/sct-cases/corrupt-then-rebuild.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+stress_write_cmd: "cassandra-stress write cl=QUORUM n=10000 -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..300300300"
+stress_read_cmd:  "cassandra-stress read cl=ONE n=10000 -mode cql3 native -rate threads=10 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1100200300 -port jmx=6868 -log interval=5"
+
+n_db_nodes: 4
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.2xlarge'
+instance_type_loader: 'c4.4xlarge'
```
Comment (roydahan) [inline test-cases/sct-cases/corrupt-then-rebuild.yaml]: remove

File: test-cases/sct-cases/corrupt-then-rebuild.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+instance_type_monitor: 't2.small'
```
Comment (roydahan) [inline test-cases/sct-cases/corrupt-then-rebuild.yaml]: remove

File: test-cases/sct-cases/destroy-data-then-repair.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# TODO: need to qualify
+test_duration: 10080
+stress_write_cmd: "cassandra-stress write cl=QUORUM n=10000 -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..300300300"
+stress_read_cmd:  "cassandra-stress read cl=ONE n=10000 -mode cql3 native -rate threads=10 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1100200300 -port jmx=6868 -log interval=5"
+
+n_db_nodes: 4
+n_loaders: 4
```
Comment (roydahan) [inline test-cases/sct-cases/destroy-data-then-repair.yaml]: Change to 2

File: test-cases/sct-cases/destroy-data-then-repair.yaml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.2xlarge'
```
Comment (roydahan) [inline test-cases/sct-cases/destroy-data-then-repair.yaml]: change to i3.large

File: test-cases/sct-cases/destroy-data-then-repair.yaml [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+instance_type_loader: 'c4.4xlarge'
```
Comment (roydahan) [inline test-cases/sct-cases/destroy-data-then-repair.yaml]: remove

File: test-cases/sct-cases/destroy-data-then-repair.yaml [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+instance_type_monitor: 't2.small'
```
Comment (roydahan) [inline test-cases/sct-cases/destroy-data-then-repair.yaml]: remove

File: test-cases/sct-cases/enospc-30mins.yaml [hunk 10]
```diff
@@ -0,0 +1,16 @@
+# TODO: need to qualify
+test_duration: 40
+
+stress_cmd: cassandra-stress write no-warmup cl=QUORUM duration=20m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -pop seq=1..10000000
+
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_db: 'c3.large' # for cassandra we'll use 'm3.large'
```
Comment (roydahan) [inline test-cases/sct-cases/enospc-30mins.yaml]: i3.large

File: test-cases/sct-cases/enospc-30mins.yaml [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+instance_type_loader: 'c4.4xlarge'
```
Comment (roydahan) [inline test-cases/sct-cases/enospc-30mins.yaml]: remove

File: test-cases/sct-cases/enospc-30mins.yaml [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+instance_type_monitor: 't2.small'
```
Comment (roydahan) [inline test-cases/sct-cases/enospc-30mins.yaml]: remove

File: test-cases/sct-cases/gce-multiple-dc-shutdown-30mins.yaml [hunk 13]
```diff
@@ -0,0 +1,11 @@
+# TODO: need to qualify
```
Comment (roydahan) [inline test-cases/sct-cases/gce-multiple-dc-shutdown-30mins.yaml]: remove this file, it's incorrect and not relevant anymore.

File: test-cases/sct-cases/google-cloud-snitch.yaml [hunk 14]
```diff
@@ -0,0 +1,14 @@
+# TODO: need to qualify
```
Comment (roydahan) [inline test-cases/sct-cases/google-cloud-snitch.yaml]: change file name to "gce-snitch-multi-dc"

File: test-cases/sct-cases/refresh-30mins-120gb.yaml [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+test_duration: 100
+
+prepare_stress_cmd: cassandra-stress write no-warmup cl=QUORUM n=10000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5
+stress_cmd: cassandra-stress mixed no-warmup cl=QUORUM duration=80m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5
+
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_db: 'c3.large'
```
Comment (roydahan) [inline test-cases/sct-cases/refresh-30mins-120gb.yaml]: i3.large

File: test-cases/sct-cases/refresh-30mins-120gb.yaml [hunk 16]
```diff
@@ -0,0 +1,23 @@
+# TODO: need to qualify
+test_duration: 100
+
+prepare_stress_cmd: cassandra-stress write no-warmup cl=QUORUM n=10000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5
+stress_cmd: cassandra-stress mixed no-warmup cl=QUORUM duration=80m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5
+
+n_db_nodes: 3
+n_loaders: 4
```
Comment (roydahan) [inline test-cases/sct-cases/refresh-30mins-120gb.yaml]: 2

File: test-cases/sct-cases/refresh-30mins-120gb.yaml [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+n_monitor_nodes: 1
+
+instance_type_db: 'c3.large'
+instance_type_loader: 'c4.4xlarge'
```
Comment (roydahan) [inline test-cases/sct-cases/refresh-30mins-120gb.yaml]: remove

File: test-cases/sct-cases/refresh-30mins-120gb.yaml [hunk 18]
```diff
@@ -0,0 +1,23 @@
...
+instance_type_monitor: 't2.small'
```
Comment (roydahan) [inline test-cases/sct-cases/refresh-30mins-120gb.yaml]: remove

File: test-cases/upgrades/rolling-upgrade.yaml [hunk 19]
```diff
@@ -0,0 +1,34 @@
+# TODO: need to qualify on GCE and AWS
```
Comment (roydahan) [inline test-cases/upgrades/rolling-upgrade.yaml]: This has pending changes in PR from @amoskong.
We'll let the changes go in to the original files and after we merge both PRs, either you or @amoskong will change this one.

File: test-cases/sct-cases/dns-cluster-5min.yaml (hunk 3 above)
Comment (fruch) [reply test-cases/sct-cases/dns-cluster-5min.yaml]: add a pipeline for it

File: test-cases/upgrades/rolling-upgrade.yaml (hunk 19 above)
Comment (fruch) [reply test-cases/upgrades/rolling-upgrade.yaml]: I've rebased based on @amoskong changes

**Comment by fruch** [comment]
@roydahan @bentsi from my POV this one is ready for merge.
and then I can move into merging all the new jenkins pipelines

**Pull request merged**